connection.go

```diff
@@ -891,19 +891,19 @@ func (c *Connection) sendBufSize() (sendBufUsage int, sendBufSize int, _ error)
 
 	var (
 		// current send buffer usage
```

Comment:
can you delete this comment and below (they are stale from when I was declaring the variables here), can probably make this block just:
```
var sendBufLenErr, sendBufLimitErr error
```

Reply:
Sure thing, done.